osu.Game/Screens/Edit/Setup/ResourcesSection.cs

```diff
@@ -40,93 +39,113 @@ private void load()
         {
             Children = new Drawable[]
             {
-                backgroundTextBox = new FileChooserLabelledTextBox(".jpg", ".jpeg", ".png")
+                backgroundChooser = new LabelledFileChooser(".jpg", ".jpeg", ".png")
                 {
                     Label = "Background",
                     FixedLabelWidth = LABEL_WIDTH,
-                    PlaceholderText = "Click to select a background image",
-                    Current = { Value = working.Value.Metadata.BackgroundFile },
                     TabbableContentContainer = this
                 },
-                audioTrackTextBox = new FileChooserLabelledTextBox(".mp3", ".ogg")
+                audioTrackChooser = new LabelledFileChooser(".mp3", ".ogg")
                 {
                     Label = "Audio Track",
                     FixedLabelWidth = LABEL_WIDTH,
-                    PlaceholderText = "Click to select a track",
-                    Current = { Value = working.Value.Metadata.AudioFile },
                     TabbableContentContainer = this
                 },
             };
 
-            backgroundTextBox.Current.BindValueChanged(backgroundChanged);
-            audioTrackTextBox.Current.BindValueChanged(audioTrackChanged);
+            if (!string.IsNullOrEmpty(working.Value.Metadata.BackgroundFile))
+                backgroundChooser.Current.Value = new FileInfo(working.Value.Metadata.BackgroundFile);
+
+            if (!string.IsNullOrEmpty(working.Value.Metadata.AudioFile))
+                audioTrackChooser.Current.Value = new FileInfo(working.Value.Metadata.AudioFile);
```

Comment:
These initial value specs change behaviour from `master`.

On `master`, if you open the dropdowns on a map that already has a background/audio file, you will start browsing in your home directory. On this branch, the initial directory is the directory where the osu! executable lies.

https://user-images.githubusercontent.com/20418176/174128390-f4d10f54-e476-4f1a-af28-fa7d9ed3683e.mp4

Seems like a pretty big UX regression.